iceoryx_examples/callbacks/README.md

```diff
@@ -0,0 +1,2 @@
+# ActiveCallSet - WORK IN PROGRESS -
```

Comment:
This file needs to be added to one of the markdown files in this folder: https://github.com/eclipse-iceoryx/iceoryx/tree/master/doc/website/getting-started/examples